### PR TITLE
Update Note for contributors

### PR DESCRIPTION
**Pull request by Artiume**
While we could always use more develops, I don't think the statement saying we don't have any contributors really fits well or looks good for us.

### CODE REVIEW

File: general/clients/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: Clients
 Clients connect your devices to your Jellyfin server and let you view your content on any supported device. You can find a list of clients below with their current development status.
 
 > [!NOTE]
-> Many clients have no active maintainers at the moment. If you interested in helping out, please see our [contribution guide](xref:contrib-index) and feel free to contact us for more information!
+> If you interested in helping out, please see our [contribution guide](xref:contrib-index) and feel free to contact us for more information!
```
Comment (nielsvanvelzen) [inline general/clients/index.md:11]:
```suggestion
> If you are interested in helping out, please see our [contribution guide](xref:contrib-index) and feel free to contact us for more information!
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
The build is failing, not the fault of this PR though 🤔

**Comment by nielsvanvelzen** [comment]
_yolomerge_

**Comment by HimbeersaftLP** [comment]
The second paragraph of the note needs to be updated to accommodate the removal of that first sentence, because it sounds very weird the way it is now.

**Pull request merged**